src/api/hydrateCharge.ts

```diff
@@ -0,0 +1,41 @@
+import { PAY_HYDRATE_CHARGE } from '../network/definitions/pay';
+import { sendRequest } from '../network/request';
+import { UNCAUGHT_HYDRATE_CHARGE_ERROR_CODE } from '../pay/constants';
+import { getPayErrorCode } from '../pay/utils/getPayErrorCode';
+import type {
+  HydrateChargeAPIParams,
+  HydrateChargeParams,
+  HydrateChargeResponse,
+} from './types';
+
+export async function hydrateCharge({
```

Comment:
@avidreder help me understand, what `hydrateCharge` is?

I read it, and it means nothing to me.

Like how a developer has suppose to use this?

Reply:
Totally agree charge hydration is more of an internal detail than a developer concern. I've renamed this function and supporting code to `buildPayTransaction`, let me know what you think.